route/build/src/remap.f90

```diff
@@ -151,6 +163,8 @@ subroutine remap_2D_runoff(runoff_data, remap_data, structHRU2seg, basinRunoff,
 
   end do   ! looping through basins in the routing layer
```

Comment:
This comment and the line 288 should be 
> looping through basins in the mapping layer

Reply:
Thanks @nmizukami -- can you implement this change?